app/workers/rapid_ready_for_decision/form526_asthma_job.rb

```diff
@@ -7,54 +7,5 @@ class Form526AsthmaJob < Form526BaseJob
     STATSD_KEY_PREFIX = 'worker.fast_track.form526_asthma_job'
 
     sidekiq_options retry: 2
-
```

Comment:
Seems like this would be ready to switch over too (remove this and update in constants). Are we keeping this file for the different retry count? Or is it to break it up more (e.g. with tests)?

Reply:
We're keeping it for the retry count. Once the asthma-related code is stable, we can update `Constants` to use `Form526BaseJob` and delete `Form526AsthmaJob`.